Cargo.toml

```diff
@@ -20,7 +20,7 @@ travis-ci = { repository = "alexcrichton/toml-rs" }
 
 [dependencies]
 serde = "1.0"
+serde_derive = "1.0"
```

Comment:
Looks like you wrote out the Deserialize impl so this is no longer used.

Reply:
Fixed, thanks!